### ISSUE TITLE
[Bug/Feature Request] Add GIF support

### ISSUE THREAD

**Issue by noirscape** [Difegue/LANraragi#257]
**LRR Version and OS**
- Arch Linux
- LRR version 0.6.8
- Docker

**Bug Details**
It's not possible to load archives containing GIF files. Attempting to do so gives the following error:

```
Image::Scale was not built with GIF support
```

Similarly, LRR is unable to generate thumbnails for these kinds of archives.

**Matching Logs**
```
[2020-02-02 21:38:00] [Auto-Plugin] [info] Executing enabled plugins on archive with id 5261af641eb79e73fde41ac83324f54ed0014b2c.
[2020-02-02 21:38:01] [Auto-Tagger] [info] Thumbnail hash invalid, regenerating.
[2020-02-02 21:38:01] [Auto-Tagger] [warn] Error building thumbnail: Image::Scale was not built with GIF support
```

**Comment by Difegue** [comment]
Should be fixed in the latest Docker nightlies due to #238 .  
Haven't got time to test those yet though so I haven't closed the issue.

**Comment by Difegue** [comment]
Tested 'n working.
